### DISCUSSION TITLE
Which ORM recommended to complement Workerman speed?

### DISCUSSION THREAD

**Discussion by cayolblake** [walkor/Workerman#779] [General]
Hello,

I wonder what's the most recommended ORM that complements the speed of Workerman and not waste any resources or introduce overhead.

@walkor would appreciate your input here buddy!

**Comment by walkor** [comment]
What I use in my project is [illuminate/database](https://github.com/illuminate/database).
I am not familiar with other ORM.

**Comment by cayolblake** [comment]
Yeah, I was hoping to see something other than illuminate. But many thanks for your reply.

**Comment by joanhey** [comment]
You can use any ORM.
That I know, it don't exist any ORM that it's optimised for persistence PHP applications.

And eloquent is not fast in normal PHP.
![image](https://user-images.githubusercontent.com/249085/175773350-470ec1d3-37ca-4ce2-a16b-6e8efa0bcaa3.png)
